### PR TITLE
LPS-178022 Cleanup TypeFacet for optional removal

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

File: modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/type/facet/portlet/TypeFacetPortletPreferencesImpl.java [hunk 1]
```diff
@@ -74,16 +76,16 @@ public String getAssetTypesString() {
 	public List<KeyValuePair> getAvailableAssetTypes(
 		long companyId, Locale locale) {
 
-		String[] assetTypes = getAssetTypesArray();
+		Set<String> assetTypes = SetUtil.fromArray(getAssetTypesArray());
```
Comment (julschong) [inline modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/type/facet/portlet/TypeFacetPortletPreferencesImpl.java:85]: Added this on the side, I can remove if unnecessary

**Comment by julschong** [comment]
ci:test:search

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test:search

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test:search

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test:search

**Comment by Kvale1733** [comment]
@julschong Regression found. Type facet widget doesn't work.

Steps to reproduce
1. Go to home page
2. Search "test"

Expected Results: The type facet shows several search types checkboxes that can be used to filter search results
Actual Results: The type facet widget does not display anything

This is how it should work:


https://user-images.githubusercontent.com/90647688/233713260-18866472-4c60-4686-9ec8-d1af180de962.mp4

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test:search

**Pull request closed without merging**